js/index.js

```diff
@@ -1,7 +1,64 @@
 // Iteration 1: Names and Input
 
+let hacker1 = "Vera"
+
+console.log("The driver's name is " + hacker1)
+
+let hacker2 = "Santi"
+
+console.log("The navigator's name is " + hacker2)
```

Comment:
Bien, pero cuando puedas usa `string interpolation`: 
console.log(`The navigator's name is ${hacker2}`)